common/config/rush/nonbrowser-approved-packages.json

```diff
@@ -602,13 +621,17 @@
       "name": "typescript",
       "allowedCategories": [ "libraries", "tests" ]
     },
+    {
+      "name": "url-loader",
+      "allowedCategories": [ "libraries" ]
+    },
     {
       "name": "webpack",
       "allowedCategories": [ "libraries", "tests" ]
     },
```

Comment:
By the alphabetization rules used elsewhere, this should be below `webpack-sources`.